lib/proxy/util.rb

```diff
@@ -11,16 +11,21 @@ class CommandTask
     # create new task and spawn new thread logging all the cmd
     # output to the proxy log. only the process' output is connected
     # stderr is redirected to proxy error log, stdout to proxy debug log
-    def initialize(acommand)
+    # command can be either string or array (command + arguments)
+    # input must be string
+    def initialize(acommand, aninput = nil)
```

Comment:
I thought `aninput` as hard to read. Is `input` a reserved value/method? If so, I think `input_value` might be easier to read.

Reply:
I wish we also renamed `acommand` to just `command` at the same time. I know it's not changed by this PR :-)